lib/utils_disk.c

```diff
@@ -196,6 +207,30 @@ np_find_name (struct name_list *list, const char *name)
   return FALSE;
 }
 
+/* Returns TRUE if name is in list */
+int
```

Comment:
Please declare this function as `bool` type, since it does return a boolean and use `true`/`false` instead of the `TRUE`/`FALSE`